girder/api/v1/system.py

```diff
@@ -347,46 +347,28 @@ def systemStatus(self, params):
         .errorResponse('You are not a system administrator.', 403)
     )
     def systemConsistencyCheck(self, params):
-        results = {}
         progress = self.boolParam('progress', params, default=False)
-        models = ('item', )
-        steps = 0
-        if progress:
-            for model in models:
-                count, changed = self.model(model).checkConsistency(
-                    stage='count')
-                steps += count
-        with ProgressContext(progress, user=self.getCurrentUser(),
-                             title='Checking system', total=steps,
-                             message='Checking system...') as ctx:
-            for model in models:
-                count, removed = self.model(model).checkConsistency(
-                    stage='remove', progress=ctx)
-                if removed:
-                    results[model+'Removed'] = removed
-            revmodels = list(models)
-            revmodels.reverse()
-            for model in revmodels:
-                count, corrected = self.model(model).checkConsistency(
-                    stage='verify', progress=ctx)
-                if count:
-                    results[model+'Count'] = count
-                if corrected:
-                    results[model+'Corrected'] = corrected
-            # TODO:
-            # * check that all files are associated with an existing item
-            # * check that all files exist within their assetstore and are the
-            #   expected size
-            # * check that all folders have a valid ancestor tree leading to a
-            #   user or collection
-            # * check that all folders have the correct baseParentId and
-            #   baseParentType
-            # * check that all groups contain valid users
-            # * check that all resources validate
-            # * for filesystem assetstores, find files that are not tracked.
-            # * for gridfs assetstores, find chunks that are not tracked.
-            # * for s3 assetstores, find elements that are not tracked.
-        return results
+        user = self.getCurrentUser()
+        title = 'Running system consistency check'
+        with ProgressContext(progress, user=user, title=title) as pc:
+            results = {}
+            results['orphansRemoved'] = self._pruneOrphans(pc)
```

Comment:
@fogleman I think he meant change this (and the next two lines similarly) to

`results['orphansRemoved'] = self._pruneOrphans(pc, ' (step 1 of 3)')`